modules/rest/openapi/openapi.yaml

```diff
@@ -6,7 +6,7 @@ info:
   license:
     name: Apache 2.0
     url: https://ignite.apache.org
-  version: 3.0.0-SNAPSHOT
+  version: 3.0.0-alpha
```

Comment:
What's this?

Reply:
Oh... this is an auto-generated file, IIRC, we must commit as soon as it generates with a new version. I committed it accidentally, but it probably needs to be committed. I will ask the guys in Slack.

Reply:
Reverted thechange